src/js/2d-board.js

```
@@ -80,7 +80,16 @@ async function messageHandler(event){
 				server.send("Game#" + gameData.id + " " + fromPTN(value));
 				setDisable2DBoard(true);
 			}
-			notate(value);
+			// Double Black Stack: white's first ply is written "2a1". PTN Ninja
+			// emits a plain "a1" for the placement, so add the leading "2"
+			// ourselves (idempotent — mirrors board.js notatePmove and
+			// server.js processGameMove). The server send above is unaffected:
+			// fromPTN ignores a placement's leading count.
+			let plyNotation = value;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)